core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -644,10 +645,18 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
         case None => gcsDAO.getGoogleProject(billingProject.googleProjectId).map(googleProject => GoogleProjectNumber(googleProject.getProjectNumber.toString))
       }
 
+      // all v2 workspaces in the specified Terra billing project will already have their own Google project number, but any v1 workspaces should store the Terra billing project's Google project number
+      workspaces <- dataSource.inTransaction { dataAccess =>
+        dataAccess.workspaceQuery.listWithBillingProject(projectName)
+      }
+      v1Workspaces = workspaces.filterNot(_.googleProjectNumber.isDefined)
```

Comment:
https://github.com/broadinstitute/rawls/blob/v2_workspaces/core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20201106_workspace_google_project_columns.xml

I think this scenario is unlikely to occur given the liquibase.  Therefore I'm inclined to remove this conditional, but it's probably not hurting anything either.

Reply:
If the Terra billing project doesn't have its Google project number stored when that liquibase changeset runs, then I believe we'd still need this conditional. I realize now I also need to update the Terra billing project with the Google project number and service perimeter name